Makefile

```diff
@@ -7,13 +7,18 @@ help: ##@other Show this help
 include ./static/tools/mk/lint.mk
 
 ifndef GOPATH
-$(error GOPATH not set. Please set GOPATH and make sure status-go is located at $$GOPATH/src/github.com/status-im/status-go. For more information about the GOPATH environment variable, see https://golang.org/doc/code.html#GOPATH)
+	$(error GOPATH not set. Please set GOPATH and make sure status-go is located at $$GOPATH/src/github.com/status-im/status-go. \
+	For more information about the GOPATH environment variable, see https://golang.org/doc/code.html#GOPATH)
 endif
 
 CGO_CFLAGS=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 GOBIN = build/bin
 GO ?= latest
 
+DCKR_IMAGE_NAME ?= status-go
```

Comment:
Nitpick, but what's the reason for saving 2 bytes of vowels in DCKR :)

Reply:
I have seen it on a few projects and didn't think about it :D But yeah, there is no need to follow this pattern.